src/plugins/dragresize/index.js

```diff
@@ -67,8 +67,26 @@ converse.plugins.add('converse-dragresize', {
             document.removeEventListener('mouseup', onMouseUp);
         }
 
+        function dragresizeOverIframeHandler (e) {
+            e.onmousedown = function () {
+                document.querySelector('iframe').style.pointerEvents  = 'none';
+            };
+            e.onmouseup = function () {
+                document.querySelector('iframe').style.pointerEvents  = 'initial';
+            };
+        }
+        
         api.listen.on('registeredGlobalEventHandlers', registerGlobalEventHandlers);
         api.listen.on('unregisteredGlobalEventHandlers', unregisterGlobalEventHandlers);
         api.listen.on('beforeShowingChatView', view => view.initDragResize().setDimensions());
+        api.listen.on('startDiagonalResize', function (e) {
+            dragresizeOverIframeHandler(e);
+        });
```

Comment:
Given that `dragresizeOverIframeHandler` doesn't reference `this`, there's no need to wrap it inside an anonymous function.

You can just do this: `api.listen.on('startDiagonalResize', dragresizeOverIframeHandler);`

Reply:
That makes sense, thanks. I'll fix it.